packages/webapp/src/app/merge.ts

```diff
@@ -54,11 +54,18 @@ function showMerge(data: {
     sanitizer: new Sanitizer(),
   });
 
+  const showBase = getConfigOption('showBase', true);
+
+  if (!showBase) {
+    document.querySelector('#nbdime-header-base')!.textContent = 'Merged';
```

Comment:
Just FYI: this header doesn't seem to be included in UI tests, I assume bc of which node is selected for screenshotting?